dist/cornerstoneTools.js

```diff
@@ -1,4 +1,8 @@
-/*! cornerstoneTools - v0.6.2 - 2015-07-31 | (c) 2014 Chris Hafey | https://github.com/chafey/cornerstoneTools */
+<<<<<<< HEAD
```

Comment:
Done. Leonardo M. Ramé
http://leonardorame.blogspot.com
      From: Erik Ziegler <notifications@github.com>
 To: chafey/cornerstoneTools <cornerstoneTools@noreply.github.com> 
Cc: Leonardo M. Ramé <martinrame@yahoo.com> 
 Sent: Monday, August 3, 2015 4:40 AM
 Subject: Re: [cornerstoneTools] Added shadow: boolean config for length and angle tools (#100)
   
In dist/cornerstoneTools.js:> @@ -1,4 +1,8 @@
> -/*! cornerstoneTools - v0.6.2 - 2015-07-31 | (c) 2014 Chris Hafey | https://github.com/chafey/cornerstoneTools */
> +<<<<<<< HEAD
Everything looks good except for this! Looks like a rebase / merge artifact. Try running grunt again and committing the result, then it looks good to merge!—
Reply to this email directly or view it on GitHub.